test/test-cases.txt

```diff
@@ -90,3 +90,9 @@
 # Text following a <br>
 {"blocks":[{"type":"unstyled","entityNodes":[{"entity":null,"styleNodes":[{"text":"Hello\nWorld!","styles":null}]}]}]}
 <p>Hello<br /> World!</p>
+
+# HTML video with iframe
+{"entityMap":{"0":{"type":"VIDEO","data":{"src":"https://www.youtube.com/embed/sFXQwk63Y9E?showinfo=0&rel=0", "width":"400", "height":"300"}}},"blocks":[{"type":"unstyled","entityNodes":[{"entity":"0","styleNodes":[{"text":"</p>","styles":null}]}]}]}
```

Comment:
Hmm, maybe it's an issue with my code. I think there's a list somewhere of self-closing tags or void elements or something. iframe should be in that list..